civis/ml/_model.py

```diff
@@ -819,6 +859,15 @@ def train(self, df=None, csv_path=None, table_name=None,
         polling_interval : float, optional
             Check for job completion every this number of seconds.
             Do not set if using the notifications endpoint.
+        validation_data : str, optional
+            Source for validation data. There are currently two options:
+            `'train'` (the default), which uses training data for validation;
+            and `'skip'`, which skips the validation step.
+        n_jobs : int, optional
+            Number of jobs to use for training and validation.
```

Comment:
Could you suggest when users might want to change this from the default, and how they should decide where to set it?

Reply:
Right now we default to 1. Do we want to do that? Why not automatically parallelize?

Reply:
What if we default to 4, so that the user always gets the benefit of parallelized cross-validation?

Reply:
Sounds reasonable to me.